cluster-install/install.sh

```diff
@@ -22,6 +22,7 @@ _EXISTING_TRIDENT_IMAGE=""
 _EXISTING_TRIDENT_ACP_ENABLED=""
 _EXISTING_TRIDENT_ACP_IMAGE=""
 _EXISTING_TRIDENT_OPERATOR_IMAGE=""
+_USER_ACCEPTED_TRIDENT_OPERATOR_UPGRADE="" # "true" if yes, "false" if no, "" if the question was never asked
```

Comment:
Currently, resource limits will be applied to the **trident-operator** when an upgrade is possible, even if the user declined said upgrade. This is used to prevent resource limits from being applied to the **trident-operator** if the user declined the upgrade.